src/backend/distributed/sql/citus--9.5-1--10.0-1.sql

```diff
@@ -70,3 +70,73 @@ DROP FUNCTION pg_catalog.mark_tables_colocated(regclass, regclass[]);
 
 #include "udfs/fix_pre_citus10_partitioned_table_constraint_names/10.0-1.sql"
 #include "udfs/worker_fix_pre_citus10_partitioned_table_constraint_names/10.0-1.sql"
+DROP FUNCTION pg_catalog.citus_dist_stat_activity CASCADE;
```

Comment:
Hmm, I find the name `citus` here a little inconvenient. `master_query_host_name` and `master_query_host_port` shows the distributed query itself (e.g., not the queries on the shards). Some information here: https://gist.github.com/onderkalaci/3c1c006dc28c692a191ef680c1b6e6ff#file-citus_dist_stat_activity-txt-L17



So, in that sense, I'd suggest two name: `distributed_query_host_name` / `distributed_query_host_port`  or  `coordinator_query_host_name` / `coordinator_query_host_port`. 

I have a slight preference for the former because on MX the distributed queries that run on the workers also show up in this view. 

 What do you think @agedemenli? Do you have any alternatives?

Reply:
Makes sense. `distributed` sounded more appropriate to me. @marcocitus ?

Reply:
yes, seems reasonable, one other alternative could be query_coordinator_hostname (as in, the coordinator of the query)